cypress/integration/slices/00-create.spec.js

```diff
@@ -1,10 +1,11 @@
 describe("Create Slices", () => {
   const name = "TestSlice";
   const lib = "slices--ecommerce"; // name of the first lib of the next project.
+  const path = "e2e-projects/next/slices/ecommerce"; // path to th library
   beforeEach(() => {
     cy.clearLocalStorageSnapshot();
     cy.cleanSliceMachineUserContext();
-    cy.task("rmrf", `${lib}/${name}`);
+    cy.task("rmrf", `${path}/${name}`);
```

Comment:
that's nice, our test are definitely creating way too many things.

Reply:
It'll still create them, this just cleans up the target first :)

Reply:
shit, we should have something to clean up after a scenario, I think that should be standard

Reply:
The cypress documentation says to clean up first and keep the data.... usually so it can be inspected later on, like if a bad model was made, I would like to keep it on disk so I can look at it

Reply:
Hmmmm i see. I guess at some point we might as well commit them then.